space_robots/Dockerfile

```diff
@@ -32,42 +32,45 @@ LABEL org.label-schema.description="Curiosity rover demo on the Space ROS platfo
 LABEL org.label-schema.vendor="Open Robotics"
 LABEL org.label-schema.version=${VERSION}
 LABEL org.label-schema.url="https://github.com/space-ros"
-LABEL org.label-schema.vcs-url="https://github.com/space-ros/docker-images"
+LABEL org.label-schema.vcs-url="https://github.com/space-ros/docker"
 LABEL org.label-schema.vcs-ref=${VCS_REF}
 
-# Disable prompting during package installation
-ARG DEBIAN_FRONTEND=noninteractive
-
 # Define a few key variables
-ARG SPACEROS_DEMO_DIR=${SRC_DIR}/spaceros_demo
+ARG DEMO_DIR=${HOME_DIR}/demos_ws
 ENV IGNITION_VERSION fortress
+ENV GZ_VERSION fortress
+
+# Disable prompting during package installation
+ARG DEBIAN_FRONTEND=noninteractive
+RUN /bin/bash -c 'echo 'debconf debconf/frontend select Noninteractive' | sudo debconf-set-selections' \
+&& sudo apt-get install -y -q
 
 # Get rosinstall_generator
 RUN sudo apt-get update -y && sudo apt-get install -y python3-rosinstall-generator
 
 # Generate repos file for demo dependencies, excluding packages from Space ROS core.
-RUN mkdir -p ${SPACEROS_DEMO_DIR}/src
-WORKDIR ${SPACEROS_DEMO_DIR}
-COPY --chown=${USERNAME}:${USERNAME} spaceros-demo-pkgs.txt /tmp/
-COPY --chown=${USERNAME}:${USERNAME} excluded-pkgs.txt /tmp/
+COPY --chown=${USERNAME}:${USERNAME} demo-pkgs.txt /tmp/
 RUN rosinstall_generator \
   --rosdistro ${ROSDISTRO} \
   --deps \
   --exclude-path ${SPACEROS_DIR}/src \
-  --exclude $(cat /tmp/excluded-pkgs.txt) -- \
```

Comment:
This line (and the copy of the excluded-pkgs.txt file above) cannot be removed or rosinstall_generator will resolve dependencies which we do not want resolved in Space ROS builds.

Reply:
So this was here because space-ros was being built again with the demos, I figured since its taken care of in the spaceros_dir it shouldn't be required in demos_dir as well?